src/components/Modal/variant/SettingsModal.scss

```diff
@@ -35,4 +35,34 @@
 .settings-modal__show-author-checkbox {
   display: block;
   margin: 1.5em 0;
+}
+
+.settings-modal__delete-button {
+  border: 0;
+  background: none;
+  outline: none;
+  width: 100%;
+
+  font-size: $theme-font-size;
+  font-weight: bold;
+  text-transform: uppercase;
+
+  padding: $default-padding $large-padding;
+
+  margin-top: $large-margin;
+  color: $color-white;
+  background: $color-mint;
```

Comment:
Maybe you can introduce another color, e.g. an orange to to indicate the danger of this action? There's an already defined `$color-orange` variable and used for the timer :)
Or you could change it in a grayish color tone and set the orange for `focus` and `hover` states as defined in line 66.

Reply:
Done